go.mod

```diff
@@ -33,6 +33,8 @@ require (
 	github.com/spf13/pflag v1.0.5
 	github.com/stretchr/testify v1.8.1
 	github.com/theupdateframework/notary v0.7.0
+	github.com/tilt-dev/fsevents v0.0.0-20200515134857-2efe37af20de
+	github.com/tilt-dev/fsnotify v1.4.8-0.20220602155310-fff9c274a375
```

Comment:
Do we know what patches are in this fork, and if they were rejected upstream? I know `fsnotify` has had some time where maintenance was slow, but I think it improved in that respect (and I _think_ we have some maintainers on it that are also maintainers for moby/moby)

Reply:
Ah, thanks! Yes, saw the comment later on, and saw that (github indicated "10" commits in the fork);

https://github.com/fsnotify/fsnotify/compare/main...tilt-dev:fsnotify:main

Pinning to a commit from upstream SGTM (short term). I'm mostly trying to avoid having 2 forks of the same dependency (as I know we have fsnotify as dependency in our tree already).

If someone has some cycles to spare to look what patches are not (yet) in upstream, we could contribute them there.

@cpuguy83 were you a maintainer on that repo? Or do I misremember that? (Otherwise I _think_ there's some familiar people on it, that we may try to reach out to to ask for a (pre-)release).

Reply:
I think I have commit access in order to get some important changes in, but I wouldn't call myself a "maintainer".

Reply:
watcher_naive relies on `SetRecursive` which doesn't exist in upstream